ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -379,44 +398,40 @@ class PropertySpec: QuickSpec {
 						expect(transformedProperty.value) == 2
 					}
 
-					describe("signal lifetime and producer lifetime") {
-						it("should return a producer and a signal which respect the lifetime of the source property instead of the read-only view itself") {
-							var signalCompleted = 0
-							var producerCompleted = 0
+					it("should return a producer and a signal which respect the lifetime of the source property instead of the read-only view itself") {
+						var signalCompleted = 0
+						var producerCompleted = 0
 
-							var property = Optional(MutableProperty(1))
-							var firstMappedProperty = Optional(property!.map { $0 + 1 })
-							var secondMappedProperty = Optional(firstMappedProperty!.map { $0 + 2 })
-							var thirdMappedProperty = Optional(secondMappedProperty!.map { $0 + 2 })
+						var property = Optional(MutableProperty(1))
```

Comment:
We are not promising these properties would _eventually_ be here when needed, but asserting that they would be gone eventually. Optional seems fine & on point.